.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Linters
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['main']
+
+  workflow_dispatch:
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install yamllint
+        run: sudo apt-get install yamllint
+
+      - name: Run yamllint
+        run: yamllint --format github $GITHUB_WORKSPACE
```

Comment:
where does `GITHUB_WORKSPACE` variable come from?

Reply:
That `GITHUB_WORKSPACE` environment variable is setup anytime one uses GitHub's `actions/checkout@v3` default action to load the gitrepo into the runner. It's the root of the git repo inside the runner. It happens from this line:
```
steps:
      - uses: actions/checkout@v3
```